x/nft/nft.pb.go

```diff
@@ -25,11 +25,18 @@ const _ = proto.GoGoProtoPackageIsVersion3 // please upgrade the proto package
 
 // Class defines the class of the nft type.
 type Class struct {
-	Id          string `protobuf:"bytes,1,opt,name=id,proto3" json:"id,omitempty"`
-	Name        string `protobuf:"bytes,2,opt,name=name,proto3" json:"name,omitempty"`
-	Symbol      string `protobuf:"bytes,3,opt,name=symbol,proto3" json:"symbol,omitempty"`
+	// id defines the unique identifier of the NFT classification, similar to the contract address of ERC721
+	Id string `protobuf:"bytes,1,opt,name=id,proto3" json:"id,omitempty"`
+	// name defines the human-readable name of the NFT classification
+	Name string `protobuf:"bytes,2,opt,name=name,proto3" json:"name,omitempty"`
+	// symbol is an abbreviated name for nft classification
+	Symbol string `protobuf:"bytes,3,opt,name=symbol,proto3" json:"symbol,omitempty"`
```

Comment:
does it make sense to store Symbol and Description on chain? It seams this can be long, and we have metadata / `Uri` for that. What do you think?